services/galley/src/Galley/Cassandra/Conversation.hs

```diff
@@ -78,8 +84,18 @@ createConversation (NewConversation ty usr acc arole name mtid mtimer recpt user
         convTeam = mtid,
         convDeleted = Nothing,
         convMessageTimer = mtimer,
-        convReceiptMode = recpt
+        convReceiptMode = recpt,
+        convProtocol = Just protocol,
+        convGroupId = groupId
       }
+  where
+    randomGroupId :: MonadIO m => m GroupId
+    randomGroupId = do
+      g <- newStdGen
+      let (len, g2) = genWord8 g
+          bs = fromShort . fst $ genShortByteString 256 g2
+      -- The length can be at most 256 bytes
+      pure . GroupId $ Blake2.hash (fromIntegral len) mempty bs
```

Comment:
What's the point of hashing here?

Reply:
Should I skip that step, i.e., should I construct the group ID like `pure . GroupId $ bs`, where `bs` is defined on line 96 above?

Reply:
Should I skip that step, i.e., should I construct the group ID like `pure . GroupId $ bs`, where `bs` is defined on line 96 above?

Reply:
Should I skip that step, i.e., should I construct the group ID like `pure . GroupId $ bs`, where `bs` is defined on line 96 above?